web-local/src/lib/components/column-profile/column-types/NumericProfile.svelte

```diff
@@ -41,13 +44,18 @@
     columnName,
     true
   );
-  $: rug = getRugPlotData(
+  $: rug = useRuntimeServiceGetRugHistogram(
     $runtimeStore?.instanceId,
     objectName,
-    columnName,
-    active
+    { columnName, priority: getPriorityForColumn("rug-histogram", active) },
+    {
+      query: {
+        select($query) {
+          return $query?.numericSummary?.numericOutliers?.outliers;
```

Comment:
I think convention is for select functions to be lambda functions, as in this example: https://tkdodo.eu/blog/react-query-data-transformations#3-using-the-select-option